llama_index/agent/react/types.py

```diff
@@ -28,8 +32,7 @@ class ActionReasoningStep(BaseReasoningStep):
     def get_content(self) -> str:
         """Get content."""
         return (
-            f"Thought: {self.thought}\nAction: {self.action}\n"
-            f"Action Input: {self.action_input}"
```

Comment:
this (and a few other code changes) were generated when I ran `make lint`. Might need someone else to rerun it locally as I may have some global black rules in some hovel of my dev environment enforcing more lax rules. If its nbd its nbd

Reply:
That's fine, I think it might have to do with your black/ruff versions? I can take another sweep at the linting